### DISCUSSION TITLE
Revision Request option

### DISCUSSION THREAD

**Discussion by Tsearo** [RetroAchievements/RAWeb#786] [Ideas]
Similar to how the Set Request option was recently added to the site I would like to see a Revision Request option with a separate view or filter when looking at requests. Total votes could be separate or shared between the two depending on feedback.

The goal would to be to hopefully increase visibility of old or simple progression sets that are in need of revisions.

**Comment by ghost** [comment]
I personally like this more than Games without Sets, sooner or later will someone make a Set for those popular games anyway while already existing Sets remain ignored when some of them really need a Revision Update.

Also allow to vote for the own created Sets just in case a Dev don't have the time finishing it.
